[BE] Modify activity details and user details

MihaiDruta opened the pull request:
[BE] Modify activity details and user details
https://trello.com/c/CLvhNFIw/68-be-modify-activity-details-and-user-details

Modify activity details and user details

Commit:
[BE] Modify activity details and user details
https://trello.com/c/CLvhNFIw/68-be-modify-activity-details-and-user-details

Modify activity details and user details

SpartakusMd commented on src/Entity/ActivityType.php:
```diff
@@ -20,14 +20,14 @@ class ActivityType
      * @ORM\Column(type="integer")
      * @Serializer\Expose()
-     * @Groups({"TypeDetails", "ActivityDetails", "ActivityCreate", "ActivityEdit"})
+     * @Groups({"TypeDetails", "ActivityCreate", "ActivityEdit"})
```
`id` should always be included. It will give the possibility to have links. Add the group back to the entitties.

Commit:
[BE] Modify activity details and user details
https://trello.com/c/CLvhNFIw/68-be-modify-activity-details-and-user-details

Restoring id's in details.
